packages/genesys-spark-components/src/components/stable/gux-button-multi/a11yManualChecklist.md

```diff
@@ -1,16 +1,18 @@
 # gux-button-multi manual accessibility testing status
 
-**Last Updated:** Wed Feb 02 2022 16:31:27 GMT-0500 (Eastern Standard Time)
+**Last Updated:** 2024-12-12T11:40:09.195Z
 | Pass | WCAG Success Criterion | Notes |
 | --- | --- | --- |
-| ✅ | [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) | - |
-| ✅ | [2.1.2 No Keyboard Trap](https://www.w3.org/WAI/WCAG21/Understanding/no-keyboard-trap.html) | - |
-| ✅ | [2.4.3 Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html) | - |
-| ✅ | [2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html) | - |
-| ✅ | [2.5.3 Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html#dfn-name) | - |
-| ✅ | [3.2.1 On Focus](https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html) | - |
-| ✅ | [3.2.2 On Input](https://www.w3.org/WAI/WCAG21/Understanding/on-input.html) | - |
-| ✅ | [3.3.1 Error Identification](https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html) | - |
-| ✅ | [3.2.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html) | - |
-| ✅ | [4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) | - |
-| ✅ | [4.1.3 Status Messages](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html) | - |
+| ❌ | [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html) | COMUI-3310 - Enter key not to be used when expanding the dropdown |
```

Comment:
Should we make issues clickable? [COMUI-3310](https://inindca.atlassian.net/browse/COMUI-3310)

Reply:
added 👍